enzy_htp/structure/ligand.py

```diff
@@ -37,16 +37,20 @@ class Ligand(NonCanonicalBase):
         conformer_coords: A List[List[Tuple[float,float,float]]] containing the coordinates of conformers.
     """
 
-    def __init__(self, residue_idx: int, residue_name: str, atoms: List[Atom], parent=None, **kwargs):
+    def __init__(self,
+                residue_idx: int,
+                residue_name: str,
+                atoms: List[Atom], 
+                parent=None, 
+                **kwargs):
         """
         Constructor for Ligand. Identical to Residue() ctor but also takes net_charge value.
         """
         NonCanonicalBase.__init__(self, residue_idx, residue_name, atoms, parent, **kwargs)
         self.rtype = chem.ResidueType.LIGAND
 
         self.bonds = kwargs.get('bonds', list())  # TODO change this to a more general represetation
```

Comment:
@chrisjurich Is this still necessary? I found mol2_io dont store connectivity into Atom()s and I'm putting a quick fix.